### ISSUE TITLE
CNN_code

### ISSUE THREAD

**Issue by UdaraN98** [MLSP-Research-Group-UOJ/MediHackUoJ_SPCup#1]
import tensorflow as tf
import numpy as np

# Load the input data
input1 = np.load('input1.npy')  # shape: (5640, 105)
input2 = np.load('input2.npy')  # shape: (5640, size)

# Reshape input1 to have 1 channel
input1 = input1.reshape((input1.shape[0], input1.shape[1], 1))

# Define the neural network architecture
input1_layer = tf.keras.layers.Input(shape=(105, 1))
input2_layer = tf.keras.layers.Input(shape=(size,))
conv_layer = tf.keras.layers.Conv1D(32, 3, activation='relu')(input1_layer)
pool_layer = tf.keras.layers.MaxPooling1D(2)(conv_layer)
flatten_layer = tf.keras.layers.Flatten()(pool_layer)
hidden_layer = tf.keras.layers.Concatenate()([flatten_layer, input2_layer])
hidden_layer = tf.keras.layers.Dense(64, activation='relu')(hidden_layer)
output_layer = tf.keras.layers.Dense(1, activation='sigmoid')(hidden_layer)

# Define the model
model = tf.keras.Model(inputs=[input1_layer, input2_layer], outputs=output_layer)

# Compile the model
model.compile(optimizer='adam', loss='binary_crossentropy', metrics=['accuracy'])

# Train the model
model.fit(x=[input1, input2], y=labels, epochs=10, batch_size=32)
